packages/graphql-language-service-types/src/index.ts

```diff
@@ -247,6 +247,16 @@ export type ContextToken = {
   style: string;
 };
 
+export type ContextTokenForCodeMirror = {
```

Comment:
`ContextToken` is only used by Codemirror, so lets combine this into one type

Reply:
Isn't `ContextToken` used inside `graphql-language-service-interface` even if you do with vscode?
https://github.com/graphql/graphiql/blob/917c3931634cf27eb31a0383ff7c15ffd902d221/packages/graphql-language-service-interface/src/getAutocompleteSuggestions.ts#L394